hibernate-core/src/main/java/org/hibernate/mapping/Constraint.java

```diff
@@ -156,6 +156,15 @@ public Column getColumn(int i) {
 		return columns.iterator();
 	}
 
+	/**
+	 * This is a duplicate method, that has been removed in v.6.0.
+	 * {@link #getColumnIterator()} is the one that will stay.
+	 *
+	 * @author TheGeekyAsian
```

Comment:
technically speaking, you were not the original author of the method, were you?

Reply:
No, I wasn’t. 

I assumed that’s the deprecation template where one has to add their names while adding a message. 

Should I remove that?

Reply:
Javadoc `@author` should only be reserved for the original author of the method IMO.

Reply:
I agree with you on that.

I'll add a new commit that drops the `@author` added.